Directory.Build.targets

```diff
@@ -4,7 +4,15 @@
     <CoverletOutputFormat>opencover</CoverletOutputFormat>
     <CoverletOutput>$(MSBuildThisFileDirectory).coverage/$(AssemblyName)/</CoverletOutput>
     <Exclude>[GraphQL.Samples.Schemas.Chat]*,[GraphQL.Samples.Server]*</Exclude>
-    <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsPackable)' != 'true'">
+    <NoWarn>$(NoWarn);CS1591;IDE1006;IDE0053;CS1998</NoWarn>
```

Comment:
CS1998 ?

Reply:
CS1998 won't let you write async methods if they don't contain await.  Which although makes sense for the main project, often causes a decrease in readability of tests. 

For example, when a delegate is `Func<ContextClass, Task<object>>` you can't write `async context => context.Widget`, but rather are required to write `context => Task.FromResult<object>(context.Widget)`.

So for tests CS1998 is disabled.